### PR TITLE
translate api standard

### PR DESCRIPTION
**Pull request by shanyi15**
fix https://github.com/PaddlePaddle/Paddle/issues/9518

### CODE REVIEW

File: doc/fluid/dev/api_doc_std_en.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+API Doc should contain the following parts(please write them in order):
+
+- Python API Definition
+
+  The definition of API
+
+- Function Description
+
+  Description of API's function. 
+  The discription includes: meaning, purpose and operation on input of API, reference and corresponding link(if any), formula(if necessary) and explanations of key variables in the formula.
```
Comment (abhinavarora) [inline doc/fluid/dev/api_doc_std_en.md]: discription -> **description**

File: doc/fluid/dev/api_doc_std_en.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+  Description of API parameters.
+  Introduce parameters one by one according to the order in API definition.
+  The introduction includes: data type, defualt value(if any), meaning, etc.
+
+- Returns
+
+  Introduction of API returned value.
+  Introduce meaning of returned value, provide corrspoding format if necessary.
+  If returned value is tuple containing multiple parameters, then introduce parameters one by one in order.
```
Comment (abhinavarora) [inline doc/fluid/dev/api_doc_std_en.md]: is tuple -> is **a** tuple

File: doc/fluid/dev/api_doc_std_en.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+- Args Description
+
+  Description of API parameters.
+  Introduce parameters one by one according to the order in API definition.
+  The introduction includes: data type, defualt value(if any), meaning, etc.
+
+- Returns
+
+  Introduction of API returned value.
+  Introduce meaning of returned value, provide corrspoding format if necessary.
```
Comment (abhinavarora) [inline doc/fluid/dev/api_doc_std_en.md]: corrspoding -> **corresponding**

File: doc/fluid/dev/api_doc_std_en.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+  If returned value is tuple containing multiple parameters, then introduce parameters one by one in order.
+
+- Raises（if any）
+
+  Abnormity, error that may occur, and possible reasons. If there are more than one possible abnormity or error, they should be listed in order.
```
Comment (abhinavarora) [inline doc/fluid/dev/api_doc_std_en.md]: Abnormity -> **abnormality**

File: doc/fluid/dev/api_doc_std_en.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+- Function Description
+
+  Description of API's function. 
+  The discription includes: meaning, purpose and operation on input of API, reference and corresponding link(if any), formula(if necessary) and explanations of key variables in the formula.
+
+- Args Description
+
+  Description of API parameters.
+  Introduce parameters one by one according to the order in API definition.
+  The introduction includes: data type, defualt value(if any), meaning, etc.
```
Comment (sidgoyal78) [inline doc/fluid/dev/api_doc_std_en.md]: defualt -> default

**Comment by shanyi15** [comment]
Done

**Pull request merged**